reference/api/settings.md

```diff
@@ -199,3 +402,992 @@ All settings will be reset to their default value.
+
+#### Example
+
+<CodeSamples id="get_sortable_attributes_1" />
+
+##### Response: `200 Ok`
+
+List the settings.
+
+```json
+[
+  "price", 
+  "author.surname"
+]
+```
+
+### Update sortable attributes
+
+<RouteHighlighter method="PUT" route="/indexes/{index_uid}/settings/sortable-attributes" />
+
+Update an index's sortable attributes list. This will re-index all documents in the index.
+
+If an attribute contains an object, you can use dot notation to set one or more of its keys as a value for this setting: `"sortableAttributes": ["author.surname"]`.
+
+[You can read more about sorting at query time on our dedicated guide.](/learn/advanced/sorting.md)
+
+#### Path parameters
+
+| Name        | Type   | Description                                                               |
+| :---------- | :----- | :------------------------------------------------------------------------ |
+| **`uid`** * | String | [`uid`](/learn/core_concepts/indexes.md#index-uid) of the requested index |
+
+#### Body
+
+An array of strings containing the attributes that can be used to sort search results at query time.
+
+[To learn more about sortable attributes, refer to our dedicated guide.](/learn/advanced/sorting.md)
+
+#### Example
+
+<CodeSamples id="update_sortable_attributes_1" />
+
+##### Response: `202 Accepted`
+
+```json
+{
+    "taskUid": 1,
+    "indexUid": "movies",
+    "status": "enqueued",
+    "type": "settingsUpdate",
+    "enqueuedAt": "2021-08-11T09:25:53.000000Z"
+}
+```
+
+You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks.md#get-one-task).
+
+### Reset sortable attributes
+
+<RouteHighlighter method="DELETE" route="/indexes/{index_uid}/settings/sortable-attributes"/>
+
+Reset an index's sortable attributes list back to its default value.
+
+#### Path parameters
+
+| Name        | Type   | Description                                                               |
+| :---------- | :----- | :------------------------------------------------------------------------ |
+| **`uid`** * | String | [`uid`](/learn/core_concepts/indexes.md#index-uid) of the requested index |
+
+#### Example
+
+<CodeSamples id="reset_sortable_attributes_1" />
+
+##### Response: `202 Accepted`
+
+```json
+{
+    "taskUid": 1,
+    "indexUid": "movies",
+    "status": "enqueued",
+    "type": "settingsUpdate",
+    "enqueuedAt": "2021-08-11T09:25:53.000000Z"
+}
+```
+
+You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks.md#get-one-task).
+
+## Stop words
+
+This route allows you to add a list of words ignored in your search queries. During a search, the stop words contained in your search query will be ignored by the sorting algorithm. When you add a common English word such as `the` to the stop words list, Meilisearch will not take it into consideration when calculating how relevant a result is.
```

Comment:
I would like to shorten this paragraph to 1–2 brief, efficient sentences. This is just a suggestion, so feel free to rewrite it!

```suggestion
Adding words to the stop words list causes them to be ignored in future search queries. Small words that appear in almost all of a dataset's documents are prime candidates for this list.
```